modules/virt-about-node-placement-virtualization-components.adoc

```diff
@@ -79,7 +79,7 @@ spec:
 [id="node-placement-hpp_{context}"]
 == Node placement in the HostPathProvisioner object
 
-You can configure node placement rules in the `spec.workload` field of the `HostPathProvisioner` object that you create when you install the hostpath provisioner.
+You can configure node placement rules for a virtual machine by using the `spec.workload` field of the `HostPathProvisioner` object that you create when you install the hostpath provisioner. Before you can delete a virtual machine from the node, you must first revert [from what to what?] the node placement [rule?] by changing the `spec.workload` field [to what?] and then wait for the `HostPathProvisioner` pod to be restored. You can then delete the virtual machine.
 
```

Comment:
I think it's this:

You can configure node placement rules by specifying `nodeSelector`, `affinity`, or `tolerations` for the `spec.workload` field of the `HostPathProvisioner` object that you create when you install the hostpath provisioner.

Before you delete a virtual machine (VM) from the node, you must first update the `spec.workload` field to another value and then wait for the `HostPathProvisioner` pod to restart. Then you can delete the VM from the node.

Reply:
@This topic is no longer used in 4.14, per Avital. Please focus on updates to the other files.